test/dynamo/test_logging.py

```diff
@@ -104,7 +104,7 @@ def test_dynamo_error(self, records):
         self.assertEqual(len(records), 1)
 
     test_aot = within_range_record_test(2, 6, aot=logging.INFO)
-    test_inductor_debug = within_range_record_test(3, 15, inductor=logging.DEBUG)
+    test_inductor_debug = within_range_record_test(2, 15, inductor=logging.DEBUG)
```

Comment:
I believe this happened because of https://github.com/pytorch/pytorch/issues/99144.